tests/New_ATS/pylib/run_tests.py

```diff
@@ -108,25 +111,30 @@ def run_test(test_config):
     generate_command = ' '.join([
         f"{test_config['python_path']} -m grpc_tools.protoc",
         f"--proto_path={test_config['test_folder']}",
-        f"--python_out={test_config['test_folder']}",
-        f"--pyi_out={test_config['test_folder']}",
-        f"--grpc_python_out={test_config['test_folder']}",
+        f"--python_out={test_config['python_client_folder']}",
+        f"--pyi_out={test_config['python_client_folder']}",
+        f"--grpc_python_out={test_config['python_client_folder']}",
         f"{test_config['test_name']}.proto"
     ])
     print ("Compiling proto file")
     run_command(generate_command)
 
     # 9. Call the TestServer() from test_folder/test_name_client.py and get the return value
-    client_py_path = test_config['test_folder'] / str(test_config['test_name'] + '_client.py')
-    run_client_command = ' '.join([
-        str(test_config['test_suite_folder'] / 'RunPythonClient.bat'),
-        str(client_py_path),
-        test_config['test_name']
-    ])
-    print ("Running python client")
-    output = run_command(run_client_command)
-    print(output)
-    FAILED += count_failed_testcases(output)
+    print(f"Running tests for all rpc's in {test_config['test_name']}")
+    for filename in os.listdir(test_config['python_client_folder']):
+        if "pb2" not in filename and filename.endswith(".py"):
```

Comment:
We can still use the  '_client.py' format. it just need to look into Python_client folder. We can call rpc_name as python_client_name. Basically for each test_config['test_name'], we have a Python_client folder inside that we have multuiple '_client.py' files to run.